articles/iot-edge/tutorial-csharp-module.md

```diff
@@ -289,7 +289,7 @@ To delete the entire resource group by name:
 
 3. To the right of your resource group in the result list, click **...** then **Delete resource group**.
 
-4. You will be asked to confirm the deletion of the resource group. Type the name of your resource group again to confirm, and then click **Delete**. After a few moments, the resource group and all of its contained resources are deleted.
+4. You will be asked to confirm the deletion of the resource group. Type the name of your resource group again to confirm, and then click **Delete**. After a few moments, the resource group and all its contained resources are deleted.
```

Comment:
Actually, the sentence was correct in its original form. Taking out the "of" does not make grammatical sense. The two options are: 

"the resource group and all of its contained resources"
"the resource group and its contained resources"

Either of those would be fine.